html/elements/script.json

```diff
@@ -603,6 +603,43 @@
                 "standard_track": true,
                 "deprecated": false
               }
+            },
+            "integrity": {
+              "__compat": {
+                "description": "<code>integrity</code>",
+                "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#normalizing-a-module-integrity-map",
+                "tags": [
+                  "web-features:import-map-integrity"
+                ],
+                "support": {
+                  "chrome": {
+                    "version_added": "127"
+                  },
+                  "chrome_android": "mirror",
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "ie": {
+                    "version_added": false
+                  },
+                  "oculus": "mirror",
+                  "opera": "mirror",
+                  "opera_android": "mirror",
+                  "safari": {
+                    "version_added": false
```

Comment:
@chrisdavidmills I've put Safari as "false" as @marcoscaceres suggested, as that's the most accurate description. At the same time, it might be useful to include slightly broader data regarding "available in webkit nightly" and "available in Safari Tech Preview" (not related to this specific feature, but in general)

Reply:
Filed https://github.com/mdn/browser-compat-data/issues/23217